src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.cs

```diff
@@ -1750,6 +1750,8 @@ public void BuildApplicationWithJavaSource (bool isRelease, bool expectedResult)
 		[Test]
 		public void BuildApplicationWithMonoEnvironment ([Values ("", "Normal", "Offline")] string sequencePointsMode)
 		{
+			const string supportedAbis = "armeabi-v7a;x86";
```

Comment:
This feels like something we should stick into `XABuildConfig.cs`, containing the `$(AndroidSupportedHostJitAbis)` value...

Though that would mean *rebuilding* e.g. `Xamarin.Android.Build.Tasks.csproj`/etc. whenever that value changes, which feels sub-optimal.

Is there some other way we can avoid hard-coding this value, so that we don't need to remember to update it when we change our ABI set?

Reply:
I didn't want to make too many changes in this code as, again, it's not relevant to the subject of this PR as well as because it's @dellis1972  and @jonathanpeppers  kingdom, so they will know better how to do it. However, yes, it should be specified in a way which makes it common for all the tests.